setup.cfg

```diff
@@ -23,3 +24,12 @@ versionfile_source = pysparkling/_version.py
 versionfile_build = pysparkling/_version.py
 tag_prefix = v
 # parentdir_prefix =
+
+[isort]
+src_paths = pysparkling,scripts
+skip_gitignore = True
+line_length = 119
+order_by_type = True
+from_first = True
+case_sensitive = True
+multi_line_output = 5
```

Comment:
To keep the current convention on import order, please use `force_alphabetical_sort_within_sections = True`.

Reply:
Done.

Reply:
I was wrong. To keep the current convention, remove `alphabetical`. 
And above, remove the line with `from_first`.

Reply:
Done

Reply:
That's not what I meant. Can you add `force_sort_within_sections`, i.e. what you had before but without the "alphabetical"?

Reply:
I did as you asked, but I don't like this now:
Before:
```python
import gzip
import logging
from io import BytesIO
```

After:
```python
import gzip
from io import BytesIO
import logging
```

I think it's a mix of `from` and `import`. I like the `from`s together and the `import`s together.

What do you think?

Reply:
Yes, that is what I want: when I come back to this code and I want to import another class from `io`, I should only need to check one place, not check first whether io was previously imported with `import` or with `from`.